west.yml

```diff
@@ -41,7 +41,7 @@ manifest:
       revision: d70aa052bc4b333e97a36b4e5832a8509b4aa417
     - name: fw-nrfconnect-mcuboot
       path: mcuboot
-      revision: 0341ae4f75e22e18aae9dfa42dd104e3352cf2b4
+      revision: 6168414cb103f97b016160396ccc88d9ed45d457
```

Comment:
I'm still new to the downstream, but I don't see that this is in a downstream branch yet: `git-branch --contains 6168414cb103f97b016160396ccc88d9ed45d457` doesn't print anything. Is this PR meant as a DNM?

Reply:
6168 is correct. You can see it here:

https://github.com/NordicPlayground/fw-nrfconnect-mcuboot/commits/master

Reply:
User error; should have been `git branch -r --contains 6168414cb103f97b016160396ccc88d9ed45d457`. Sorry about that.

Reply:
np